src/components/DonationWidget/components/DonationAddressForm.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+const DonationAddressForm = ({
```

Comment:
In order to keep the same pattern from other components, I'd suggest declaring prop types as following

```suggestion
interface Props {
  defaultValues: {
    address: string;
    city: string;
    zipCode: string;
    country: string;
  };
  onSubmit: (e: React.ChangeEvent<HTMLFormElement>) => void;
};

const DonationAddressForm: React.FC<Props> = ({defaultValues,
  onSubmit}) => {
 ....
}
```

Reply:
I will say you are totally on point in regards keeping the consistency. _I will argue to use types instead interface, but if this is the approach we have lets proceed with it and then we can challenge if necessary_
_I didn't know about this feature on Github_

Reply:
I won't commit from here to make a commit to address this on all components I have introduced.